topbeat/docs/reference/configuration/topbeat-options.asciidoc

```diff
@@ -26,8 +26,10 @@ topbeat:
     filesystem: true
 
     # cpu usage per core, by default is false
-    cpu_per_core: false
+    core: false
```

Comment:
yes, it is, as the CPU details per core are placed under the `type=core` documents instead of `type=system` document.s